src/main/resources/config.yml

```diff
@@ -9,7 +9,22 @@ database:
   password: ${GRAPHENEDB_BOLT_PASSWORD}
 
 amazonaws:
-  from: ${FROM_EMAIL:-""}
+  fromEmailSES: ${FROM_EMAIL:-""}
+
+emailConfig:
+  CREATE_ORG:
+    subject: "Thank you joining our program"
+    resource: "email-templates/org-join"
+  CREATE_SCANNABLE:
+    subject: "Thank you for creating a scannable"
```

Comment:
scannable is an internal term, it's not part of what we are communicating to users. we should have official copy eventually, but for now, just replace scannable with "QR Code". we'll update it later